### ISSUE TITLE
Checked menu items not read correctly by Narrator/NVDA on Windows

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5833]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.x
    OS Version      : Windows-10
    R Version       : Any

### Steps to reproduce the problem
- turn on Narrator or NVDA (probably JAWS, too, but didn't try)
- start RStudio desktop IDE
- Main Menu: Debug / On Error
- Move focus to checked menu item (e.g. "Error Inspector")

### Describe the problem in detail
- Screen reader says something like "Error Inspector Alt+E"
- No indication is given that the item has a checkmark

### Describe the behavior you expected
- Screen reader should include "checked" when focus put on a checked menu item
- Compare with behavior in a standard Windows program such as notepad.exe, where it will do this (e.g. View / Status Bar)
- Works correctly on macOS / VoiceOver

This is a Qt bug, repros in Qt Creator, and in various Qt example programs. Opened https://bugreports.qt.io/browse/QTBUG-80551

**Comment by gtritchie** [comment]
Note: fixed in Qt 5.14.1

**Comment by gtritchie** [comment]
Should be fixed if we switch to Qt 5.14.1 or above.

**Comment by gtritchie** [comment]
Verified fixed. I check with NVDA, Narrator, and JAWs on Windows-11, and VoiceOver on Mac, and in all cases checkable menu items were properly described.
